Mathlib/FieldTheory/KummerExtension.lean

```diff
@@ -146,6 +146,10 @@ theorem pow_ne_of_irreducible_X_pow_sub_C {n : ℕ} {a : K}
     Nat.cast_injective.eq_iff] at hq
   exact hm' ((mul_eq_right₀ (mul_ne_zero_iff.mp hn).2).mp hq)
 
+/--Let `p` be a prime number. Let `F` be a field.
+Let `t ∈ F` be an element which does not have a `p`th root in `F`.
+Then the polynomial `x ^ p - t` is irreducible over `F`.-/
+@[stacks 09HF]
```

Comment:
Note that we don't require char K = p here.

Reply:
Thanks! Added in the comments.